CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 ----------
 
+[3.2.0] - 2022-11-30
+--------------------
+Changed
+~~~~~~~
+* Add event_metadata parameter to EventBusProducer send API
```

Comment:
```suggestion
* Add event_metadata parameter to EventBusProducer send API. **Note:** This parameter cannot be used until the caller also upgrades to a version of openedx-events that supports this parameter.
```
...and we probably want a matching comment in the openedx-events changelog about updating implementation libraries.

Reply:
Sorry, just realized that this *is* openedx-events. Updating suggestion...

Reply:
I adopted the idea of the suggestion but used somewhat different wording that I think is more accurate. LMK what you think.

Reply:
Yeah, seems fine.